components/measurement/nettests/HTTPHeaderFieldManipulation.js

```diff
@@ -18,7 +18,7 @@ export const HttpHeaderFieldManipulationDetails = ({ measurement, render }) => {
       isFailed = false
     }
   })
-  const headerDiff = testKeys.tampering.header_name_diff
+  // const headerDiff = testKeys.tampering.header_name_diff
```

Comment:
I suggest to remove code we don't need rather than commenting it out. We have git history and we are doing a great job w/ commits, so it shan't be a problem in the future to go back and ask our past self a hand.